packages/binding.template/src/templating.ts

```diff
@@ -324,46 +332,46 @@ export class TemplateBindingHandler extends AsyncBindingHandler {
     var value = this.value
     var options = unwrap(value)
     var shouldDisplay = true
-    var templateComputed = null
-    var elseChainSatisfied = domData.get(element, 'conditional').elseChainSatisfied
-    var templateName
+    let templateComputed: string | Computed<any> | null = null;
+    var elseChainSatisfied = domData.get(element, 'conditional').elseChainSatisfied;
+    var templateName;
 
     if (typeof options === 'string') {
-      templateName = value
-      options = {}
+      templateName = value;
+      options = {};
     } else {
-      templateName = options.name
+      templateName = options.name;
 
       // Support "if"/"ifnot" conditions
       if ('if' in options) {
-        shouldDisplay = unwrap(options.if)
+        shouldDisplay = unwrap(options.if);
       }
 
       if (shouldDisplay && 'ifnot' in options) {
-        shouldDisplay = !unwrap(options.ifnot)
+        shouldDisplay = !unwrap(options.ifnot);
       }
     }
 
     if ('foreach' in options) {
       // Render once for each data point (treating data set as empty if shouldDisplay==false)
-      var dataArray = (shouldDisplay && options.foreach) || []
-      templateComputed = renderTemplateForEach(templateName || element, dataArray, options, element, bindingContext, this.completeBinding)
+      var dataArray = (shouldDisplay && options.foreach) || [];
+      templateComputed = renderTemplateForEach(templateName || element, dataArray, options, element, bindingContext, this.completeBinding);
 
-      elseChainSatisfied((unwrap(dataArray) || []).length !== 0)
+      elseChainSatisfied((unwrap(dataArray) || []).length !== 0);
```

Comment:
it'd be nice if the style guide here highlighted and ideally removed unnecessary semicolons

Reply:
I've already prepared prettier here. It's configured to maintain the existing style: https://github.com/Auge19/tko/tree/add_prettier

Personally, I would prefer globaly ending semicolons.

Reply:
Done in https://github.com/Auge19/tko/pull/4 for this file. Some unneeded semicolons was already present, prettier will be the solution in the future.